src/language/primitives.rkt

```diff
@@ -54,7 +54,7 @@
 (define-predicate IntxInt->Int-primitive? IntxInt->Int-Primitive)
 
 (define-type ->Int-Primitive (U 'read-int))
-(define-type Int->Unit-Primitive (U 'print-int))
+(define-type Int->Unit-Primitive (U 'print-int 'increment-int))
```

Comment:
This type isn't really sensible in our type system. For instance 2++ is type safe but would generate bad code.

Reply:
Done.